ibis/backends/omniscidb/tests/test_operations.py

```diff
@@ -84,7 +84,7 @@ def test_join_diff_name(awards_players, batting):
         .materialize()
         .execute()
     )
-    assert df.size == 80
+    assert df.size == 70
```

Comment:
good point. not sure why it is returning a different result. I saw this before (in someone's PR) but never investigated that.

Reply:
thanks @xmnlab